styles/style.css

```diff
@@ -163,12 +206,26 @@ p {
 
 .article {
   margin-bottom: 1rem;
+  display: flex;
+  justify-content: space-between;
+
+}
+
+section article{
+  border: solid 2px var(--grey-light);
+  width: 45%;
+  padding: 15px;
 }
 
 .article__title {
   margin-bottom: 0.5rem;
   font-size: 1rem;
   font-weight: 700;
+
+}
+.article__content {
+padding: 9px 9px;
```

Comment:
Some of the indentation in this file isn't quite right. If you install the Prettier extension in VS Code and enable 'Format on Save' in your VS Code settings it should fix all the indentation for you. Let me know if you need any help!

Reply:
Your right! I've  cleaned up about 7 lines :)